dali/test/python/test_fw_iterators.py

```diff
@@ -1509,3 +1509,29 @@ def test_pytorch_feed_ndarray():
     out = pipe.run()[0]
     torch_tensor = torch.empty((1), dtype=torch.int8, device = 'cpu')
     assert_raises(AssertionError, feed_ndarray, out, torch_tensor, glob="The element type of DALI Tensor/TensorList doesn't match the element type of the target PyTorch Tensor:")
+
+# last_batch_policy type check
+def check_iterator_build_error(ErrorType, Iterator, *args, **kwargs):
+    batch_size = 4
+    num_gpus = 1
+    pipes, data_size = create_pipeline(lambda gpu: create_coco_pipeline(batch_size=batch_size, num_threads=4, shard_id=gpu, num_gpus=num_gpus,
+                                                                        data_paths=data_sets[0], random_shuffle=True, stick_to_shard=False,
+                                                                        shuffle_after_epoch=False, pad_last_batch=False), batch_size, num_gpus)
+    assert_raises(ErrorType, Iterator, pipes,
```

Comment:
How about adding error string here as well to verify that the error is indeed the one expected (especially in case of same changes in the future)?

Reply:
Good idea. Will do